### DISCUSSION TITLE
Drones and Maximum Flying Time

### DISCUSSION THREAD

**Discussion by 5th-Domain** [PygmalionOfCyprus/cmo-db-requests#4217] [Q&A] [answered]
DB: DB3K_504

Currently in the database some of the drone have a maximum flying time (Pilot Fatigue) the same as manned aircraft is this correct? If this is not correct do you want the ones i can find reporting in here or Matrix tech support.

Thanks

**Comment by PygmalionOfCyprus** [comment, marked as answer]
Depends on the UAV. A fair few of them have endurance numbers plugged in because of manufacturer specs as to max endurance (not pilot fatigue, but perhaps engine life, etc.). It's possible the ones you're referring to have the numbers in error, but I'd expect it to be the UAV endurance.

**Reply by 5th-Domain** [reply]
Thanks for the reply that makes sense and fits with what i was seeing then.
